app/main.py

```diff
@@ -1,6 +1,29 @@
+import os
+import requests
+import dotenv
+dotenv.load_dotenv()
+
+
+FILTERING = "Paris"
+
+API_KEY = os.getenv("API_KEY")
+
+URL = "https://api.weatherapi.com/v1/current.json"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    params = {"key": API_KEY, "q": FILTERING}
+    res = requests.get(URL, params=params).json()
+    if res.get("error") is not None:
+        print(res.get("error"))
+        return
+    print(
+        f"Country: {res.get('location').get('country')};\n"
```

Comment:
not good to write .get().get()
Because in this case NoneType error possible add default as dict in all places